superset/assets/package.json

```diff
@@ -150,13 +152,15 @@
     "react-addons-test-utils": "^15.6.2",
     "react-test-renderer": "^15.6.2",
     "redux-mock-store": "^1.2.3",
-    "sinon": "^4.0.0",
-    "style-loader": "^0.18.2",
+    "//": "known minor issues in >5.0",
```

Comment:
@mistercrunch what are these entries with double backslashes?

Reply:
Got this as a suggested way to squeeze comments into JSON:
https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install

This works with `yarn` but not the current version `npm`. I'm pretty sure npm used to allow this but not 100%. On Slack I mentioned to @GabeLoins that this may be a good way to force people to use `yarn` as a result, since we now use `yarn.lock`